rust/processor/src/processors/econia_processor.rs

```diff
@@ -796,11 +804,14 @@ impl ProcessorTrait for EconiaTransactionProcessor {
             }
             // Index transaction write set.
             let info = &txn.info.as_ref().expect("No transaction info");
+            let market_accounts_type_string = format!("{address}::{}::{}", market_accounts_type.address, market_accounts_type.module, market_accounts_type.name);
```

Comment:
Did you cargo check this or format on save?

I'm pretty sure this statement is a compiler error

Again, as I suggested in https://github.com/econia-labs/aptos-indexer-processors/pull/19#discussion_r1396145106, how about simply replacing the statement where `market_accounts_type` is constructed, since it isn't used anywhere else?